auto_backup/static/description/no_index.html

```diff
@@ -0,0 +1,100 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan">Automated backups</h2>
+            <h3 class="oe_slogan">A tool for all your back-ups, internal and external!</h3>
+        </div>
+	<div class="oe_span6">
+            <div class="oe_demo oe_picture oe_screenshot">
+                    <img src="overview.png">
+            </div>
+        </div>
+	<div class="oe_span6">
+            <p class="oe_mt32">
+		Keep your Odoo data safe with this module. Take automated back-ups, remove them automatically 
+		and even write them to an external server through an encrypted tunnel.
+		You can even specify how long local backups and external backups should be kept, automatically!
+            </p>
+            <div class="oe_centeralign oe_websiteonly">
+                <a href="http://www.openerp.com/start?app=mail" class="oe_button oe_big oe_tacky">Start your <span class="oe_emph">free</span> trial</a>
```

Comment:
I'd prefer a markdown readme. Simpler to maintain. This link for example does not seem very useful (and is `odoo.com` nowadays). Not a blocker IMO though.